src/index.html

```diff
@@ -5,10 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="nav__container">
+      <div class="logo__container">
+        <a href="#home" class="logo__container___link">
+          <img src="./images/logo.png" alt="MOYO logo">
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__item__link is-active" href="#">APPLE</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item__link" href="#">SAMSUNG</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__item__link" href="#">SMARTPHONES</a>
```

Comment:
```suggestion
            <a class="nav__item-link" href="#">SMARTPHONES</a>
```

Reply:
You can read more [here](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en)

Reply:
fixed all)